dbms/src/Flash/Coprocessor/DAGDriver.cpp

```diff
@@ -112,6 +113,8 @@ template <DAGRequestKind Kind>
 void DAGDriver<Kind>::execute()
 try
 {
+    if (context.getTMTContext().checkShuttingDown())
+        throw TiFlashException("TiFlash server is terminating", Errors::Coprocessor::Internal);
```

Comment:
I wonder if tiflash return this error, can tidb internally retry the query by re-dispatching the query to another tiflash instance?

Reply:
I checked the code and found that `BatchCop` supports retries when getting this error, but `Cop` does not. So I removed these two lines of code.

Reply:
> BatchCop supports retries when getting this error, but Cop does not

What about `MPP`?